app/models/row_to_ni_duty_calculator.rb

```diff
@@ -40,7 +40,7 @@ def options
 
       footnote_suffix ||= I18n.t("row_to_ni_measure_type_footnotes_suffixes.#{option.category}.#{option.source}").html_safe
 
-      option.footnote = option.footnote.concat(footnote_suffix).html_safe
+      option.footnote_suffix = footnote_suffix
```

Comment:
If this is doing the assignment, this is probably the best place to be deciding if the suffix is safe?